doc/language/globals.md

```diff
@@ -10,74 +10,75 @@ To use the module:
+|  Variable   |         \English         |  Initially   | Read-Only | Set By                                         |
+|:-----------:|:------------------------:|:------------:|:---------:|------------------------------------------------|
+| <tt>$!</tt> |   <tt>$ERROR_INFO</tt>   | <tt>nil</tt> |   Yes.    | Kernel#raise, to Exception object.             |
+| <tt>$@</tt> | <tt>$ERROR_POSITION</tt> | <tt>nil</tt> |   Yes.    | Kernel#raise, to array of backtrace positions. |
 
 ### Pattern Matching
 
-| Variable      | English             | Contains                                         |
-|---------------|---------------------|--------------------------------------------------|
-| `$~`          | `$LAST_MATCH_INFO`  | MatchData object; set by matcher method.         |
-| `$&`          | `$MATCH`            | Matched substring; set by matcher method.        |
-| `` $` ``      | `$PRE_MATCH`        | Substring left of match; set by matcher method.  |
-| `$'`          | `$POST_MATCH`       | Substring right of match; set by matcher method. |
-| `$+`          | `$LAST_PAREN_MATCH` | Last group matched; set by matcher method.       |
-| `$1`          |                     | First group matched; set by matcher method.      |
-| `$2`          |                     | Second group matched; set by matcher method.     |
-| <tt>$_n_</tt> |                     | <i>n</i>th group matched; set by matcher method. |
+|         Variable          |          \English          |  Initially   | Read-Only | Set By                                                       |
+|:-------------------------:|:--------------------------:|:------------:|:---------:|--------------------------------------------------------------|
+|        <tt>$~</tt>        | <tt>$LAST_MATCH_INFO</tt>  | <tt>nil</tt> |    No.    | Matcher method: to MatchData object or <tt>nil</tt>.         |
+|        <tt>$&</tt>        |      <tt>$MATCH</tt>       | <tt>nil</tt> |    No.    | Matcher method: to matched substring or <tt>nil</tt>.        |
+|        <tt>$`</tt>        |    <tt>$PRE_MATCH</tt>     | <tt>nil</tt> |    No.    | Matcher method: to substring left of match or <tt>nil</tt>.  |
+|        <tt>$'</tt>        |    <tt>$POST_MATCH</tt>    | <tt>nil</tt> |    No.    | Matcher method: to substring right of match or <tt>nil</tt>. |
+|        <tt>$+</tt>        | <tt>$LAST_PAREN_MATCH</tt> | <tt>nil</tt> |    No.    | Matcher method: to last group matched or <tt>nil</tt>.       |
+|        <tt>$1</tt>        |                            | <tt>nil</tt> |    No.    | Matcher method: to first group matched or <tt>nil</tt>.      |
+|        <tt>$2</tt>        |                            | <tt>nil</tt> |    No.    | Matcher method: to second group matched or <tt>nil</tt>.     |
+|       <tt>$_n_</tt>       |                            | <tt>nil</tt> |    No.    | Matcher method: to <i>n</i>th group matched or <tt>nil</tt>. |
```

Comment:
These variables are wrappers of `$~` attributes, and cannot be assigned directly.